docs/key-value-storage.md

```diff
@@ -0,0 +1,133 @@
+---
+id: kv
+title: KV Storage
+sidebar_label: KV Storage
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+defaultValue="js"
+values={[
+{label: 'Javascript', value: 'js'},
+{label: 'Python', value: 'py'},
+{label: 'PHP', value: 'php'},
+]}
+
+>
+
+Your Nimbella account has been provisioned with Key-Value Storage and Object Storage. In this document, we are going to cover Key-Value Storage.
+
+A Key-Value Store, or KV for short, acts as a database where you create keys and apply values to those keys.
+
+The following `nim` command will show the current contents of the KV. The command will not show any output if the KV is empty.
+
+```
+nim key-value list
+```
+
+This document divides into these parts:
+
+- Conceptual Overview
+- Using a Key-Value Storage inside actions using a Nimbella SDK
+  - The `nodejs` SDK
+  - The `php` SDK
+  - The `python` SDK
+- Common Redis commands
+
+### Conceptual Overview
+
+A KV behaves like a database where you can create keys, apply values to those keys, and then call on them to extract the data that you need.
+
+In this visual example, you can store user info, such as their City, State, and Country, and apply the correct values to them. This allows you to keep track of data and request the data when needed.
+
+![](https://github.com/JamieDawson/docs/blob/10b-two-types-storage-kv/static/img/kv-store-redis-example.png)
```

Comment:
this is referring to your fork. instead use a relative link `./docs/...` so that it works for anyone's fork.